packages/react/src/components/date-picker/date-picker.tsx

```diff
@@ -432,6 +432,17 @@ export const Datepicker = forwardRef(({
                 dateInputRef.current?.setOpen(false);
                 calendarButtonRef.current?.focus();
                 break;
+            case 'Enter':
+            case ' ':
+                if (dateInputRef.current?.isCalendarOpen()) {
+                    event.stopPropagation();
+                    dateInputRef.current?.setOpen(false);
+
+                    if (onCalendarClose) {
+                        onCalendarClose();
+                    }
```

Comment:
Tu peux faire ça ici pis aux 3 autres places plus bas!
```suggestion
                    onCalendarClose?.();
```